daemon/gdbus-util.c

```diff
@@ -141,3 +141,18 @@ gdbus_put_system_connection (void)
 {
   g_clear_object (&g_dbus_sys_conn);
 }
+
+/**
+ * @brief Common function to initailize the DBus module.
+ */
+void
+gdbus_initialize (void)
+{
+  GError *err = NULL;
```

Comment:
How about `g_autoptr (GError) = NULL;` for more short code?

Reply:
For future work, I am not sure another code for initializing ml-agent module which requires gerror ptr. Let's keep this in this time.